### PR TITLE
Camlp5 logger syntax extension 0.4.2

### PR DESCRIPTION
**Pull request by Kakadu**
It will not be very popular but it is a dependency for stuff that I actually want to pulish in opam.

### CODE REVIEW

**Comment by Kakadu** [comment]
Mac is failing but it is temporary inconvenience. 

Ready to review.

File: packages/logger/logger.0.4.2/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+
+name: "logger-p5"
```
Comment (mseri) [inline packages/logger/logger.0.4.2/opam]: The `name` and `version` fields are redundant, `opam` will replace them with what infers from the path.
Moreover, is the name `logger` (as in the path) or `logger-p5` (as here)?

Comment (Kakadu) [reply packages/logger/logger.0.4.2/opam]: I made a transition to `logger-p5`. Thanks

File: packages/logger-p5/logger-p5.0.4.2/opam [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+"""
+
+maintainer: "kakadu.hafanana@gmail.com"
+authors: "https://github.com/dboulytchev"
+homepage: "https://github.com/dboulytchev/logger"
+bug-reports: "https://github.com/dboulytchev/logger/issues"
+
+depends: [
+  "ocaml" { >= "4.07.0" }
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/logger-p5/logger-p5.0.4.2/opam]:
```suggestion
  "ocamlfind" {build}
```

**Comment by kit-ty-kate** [comment]
Sounds ok. Thanks

**Pull request merged**